src/details/ArborX_KDOP.hpp

```diff
@@ -23,7 +23,7 @@
 
 namespace ArborX
 {
-namespace Details
+namespace DetailsButCannotBeCalledLikeThat
```

Comment:
@aprokop I am fine reverting the changes to that file.  Just demonstrating this works as intended.